courses-app/src/app/shared/components/button/button.component.html

```diff
@@ -0,0 +1,4 @@
+<div class="button">
```

Comment:
my personal opinion is that we could use host element as a wrapper to apply styles instead of adding additional html elements